lib/Resource/BaseWorkOSResource.php

```diff
@@ -21,9 +21,9 @@ private function __construct()
     /**
      * Creates a Resource from a Response.
      *
-     * @param \WorkOS\Resource\Response $response
+     * @param Response $response
      *
-     * @return \WorkOS\Resource\*
+     * @return static
```

Comment:
This is the main phpdoc fix: it tells the return type is the type of the called class.